### DISCUSSION TITLE
How could I model such ManyToMany relationships

### DISCUSSION THREAD

**Discussion by ghostoy** [mikro-orm/mikro-orm#5961] [Q&A]
I have two tables:

```typescript
class A {
  @PrimaryKey()
  id: number;

  @Property()
  join_id: number;
}

class B {
  @PrimaryKey()
  id: number;

  @Property()
  join_id: number;
}
```

A and B make a many-to-many relationship, but they join on column `join_id` without a pivot table. How could I model such relationship using MikroORM?

**Comment by B4nan** [comment]
That's not a m:n relation, you need a pivot table for that. Also, relations need to point to a PK, not to just any column. 

Overall, I don't think you can map this to a relation here. I guess a virtual to-many relation could be a way around, but that is currently also not supported.
